apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -934,6 +934,13 @@ export class ApiModelGenerator {
 
       parentApiItem.addMember(apiTypeAlias);
     }
+
+    if (
+      ts.isTypeAliasDeclaration(astDeclaration.declaration) &&
+      ts.isTypeLiteralNode(astDeclaration.declaration.type)
```

Comment:
It's not clear to me from line 940 which type aliases will have their children processed and which will not. I think a comment providing some supported vs not supported examples could go a long way here.